src/pages/rental/rental.ts

```diff
@@ -62,18 +62,63 @@ export class RentalPage {
    * Pushes page on nav to allow users to finish renting the items.
    */
   onContinue() {
-    this.navCtrl.push(RentalDetailsPage);
+    let items;
+    this.items.take(1).subscribe(i => items = i.rentals);
+
+    // Remind user to scan items if rental is empty
+    if (!Object.keys(items).length) {
+      this.alertEmptyRental();
+    } else {
+      this.navCtrl.push(RentalDetailsPage);
+    }
   }
 
   /**
    * Returns items by calling the api for each item in the list of items to
    * return. Pops the nav when done.
    */
   onReturn() {
-    // Set return date to today in MySQL date format
-    const returnDate = new Date().toISOString().substring(0, 10);
-    this.layoutActions.showLoadingMessage(LoadingMessages.returningItems);
-    this.itemsActions.returnItems(returnDate);
+    let items;
+    this.items.take(1).subscribe(i => items = i.rentals);
+
+    // Remind user to scan items if rental is empty
+    if (!Object.keys(items).length) {
+      this.alertEmptyRental();
+    } else {
+      // Set return date to today in MySQL date format
+      const returnDate = new Date().toISOString().substring(0, 10);
+      this.layoutActions.showLoadingMessage(LoadingMessages.returningItems);
+      this.itemsActions.returnItems(returnDate);
+    }
+  }
+
+  /**
+   * Creates an alert to warn the user that the rental is empty and prevent him
+   * or her to make an empty rental.
+   */
+  alertEmptyRental() {
+    let alert = this.alertCtrl.create({
+      title: 'No items in rental',
```

Comment:
To be consistent with the description of the alert, I think you should change "rental" in the title to `${this.action.toLowerCase()}`, unless I'm guessing wrong about the value of `this.action`.